iac-extensions/arm/src/test/resources/checks/AnonymousAccessToResourceCheck/Microsoft.DataFactory_factories_linkedservices.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "type": "Microsoft.DataFactory/factories/linkedservices",
+      "apiVersion": "2018-06-01",
+      "name": "Sensitive",
+      "properties": {
+        "type": "Web",
+        "typeProperties": {
+          "authenticationType": "Anonymous"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.DataFactory/factories/linkedservices",
+      "apiVersion": "2018-06-01",
+      "name": "Compliant",
+      "properties": {
+        "type": "Web",
```

Comment:
There should be a check for each type, see `ManagedIdentityCheckTest` for inspiration

Reply:
Implemented for JSON; still we don't yet support `Noncompliant` comment in `BicepVerifier#verifyContent`. Probably, we should add it in the future